source/dofs/dof_handler.cc

```diff
@@ -17,6 +17,7 @@
 
 #include <deal.II/base/geometry_info.h>
 #include <deal.II/base/memory_consumption.h>
+#include <deal.II/base/mpi.templates.h>
```

Comment:
We need this because of:
https://github.com/dealii/dealii/blob/538ef0012a8489116311fd4302f6ca03c4d842da/source/dofs/dof_handler.cc#L1744-L1746

Reply:
Else you get an undefined reference? If yes, I would suggest to instantiate the function for `unsigned short int`.

Reply:
Let's leave it like this to be independent of the actual `active_fe_index_type`.